ydb/core/testlib/test_client.h

```diff
@@ -24,6 +24,7 @@
 #include <ydb/core/protos/kesus.pb.h>
 #include <ydb/core/protos/table_service_config.pb.h>
 #include <ydb/core/protos/console_tenant.pb.h>
+#include <ydb/core/protos/flat_tx_scheme.pb.h>
```

Comment:
Is this necessary? What symbol defs from `flat_tx_scheme.pb.h` this include require?
I suggest moving this include line to `test_client.cpp`.

Reply:
This include is used for return type `NKikimrScheme::TEvLoginResult` of `Login` function